Dockerfile

```diff
@@ -1,8 +1,12 @@
 FROM public.ecr.aws/amazonlinux/amazonlinux:2022
 
 RUN yum -y update
-RUN yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
-RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "pgbouncer_1_15_0" && \
+RUN yum install -y wget && \
+    yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
```

Comment:
just append `wget` after gcc. It doesn't need to be a separate call to `yum install`